components/formats-bsd/test/loci/formats/utests/FileStitcherTest.java

```diff
@@ -193,4 +212,43 @@ public void testStitch(Integer[] dims) throws IOException, FormatException {
     check(pattern, filenames.toArray(new String[filenames.size()]), dims);
   }
 
+  @Test
+  public void testUnderlyingReaders() throws IOException, FormatException {
+    FakeReader reader = new FakeReader();
+    FileStitcher fs = new FileStitcher(reader);
+    assertNotNull(fs.getUnderlyingReaders());
+    fs.setId("test_z<0-2>.fake");
+    assertNotNull(fs.getUnderlyingReaders());
```

Comment:
I assume the underlying readers should have been expanded from a single reader to 3 fake readers in that case? Should this also be covered in the unit test?